resources/dicts/patrols/beach/med/any.json

```
@@ -2940,5 +2940,156 @@
+                "exp": 20,
+                "weight": 10,
+                "herbs": ["random_herbs"],
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["r_c"],
+                        "mutual": true,
+                        "values": ["dislike", "jealous"],
+                        "amount": -10
+                    }
+                ]
+            },
+            {
+                "text": "Patiently, r_c listens to p_l as the healer describes {PRONOUN/p_l/poss} troubles. r_c can't make the plants healthy again, or repair the damage from the storm, but {PRONOUN/r_c/poss} compassion for p_l makes the day seem so much more hopeful. Sometimes medicine cats need to vent too.",
+                "exp": 20,
+                "weight": 20,
+                "can_have_stat": ["r_c"],
+                "stat_trait": [
+                	"compassionate",
+                	"empathetic",
+                	"faithful",
+                	"loving",
+                	"patient",
+                	"responsible",
+                	"thoughtful"
+            	],
+                "herbs": ["random_herbs"],
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["r_c"],
+                        "mutual": true,
+                        "values": ["dislike", "jealous"],
+                        "amount": -10
+                    },
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["r_c"],
+                        "mutual": true,
+                        "values": ["respect", "comfort", "trust"],
+                        "amount": 5
+                    }
+                ]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "r_c splashes water from a puddle at p_l. p_l turns around and hisses at r_c that {PRONOUN/p_l/subject} {VERB/p_l/don't/doesn't} have time for kit games! The two continue their patrol, but p_l's sour mood distracts them from any usable herbs they could find.",
+                "exp": 0,
+                "weight": 20,
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["r_c"],
+                        "mutual": true,
+                        "values": ["respect", "comfort", "trust"],
+                        "amount": -5
+                    }
+                ]
+            },
+            {
+                "text": "r_c can't think of anything to be able to help p_l's mood, and as the patrol continues, p_l's foul mood infects r_c, making {PRONOUN/r_c/object} grumpy and irritable as well, until the patrol is called off.",
```

```suggestion
                "text": "r_c can't think of anything to be able to help p_l's mood. As the patrol continues, p_l's foul mood infects r_c, making {PRONOUN/r_c/object} grumpy and irritable as well, until the patrol is called off.",
```
second sentence still clunky, consider revising.